bigchaindb/backend/mongodb/query.py

```diff
@@ -348,9 +366,12 @@ def get_new_blocks_feed(conn, start_block_id):
 
 @register_query(MongoDBConnection)
 def text_search(conn, search, *, language='english', case_sensitive=False,
-                diacritic_sensitive=False, text_score=False, limit=0):
+                diacritic_sensitive=False, text_score=False, limit=0, table=None):
+    if table is None:
```

Comment:
I think it would be cleaner and less error prone to make table a required parameter or at least change the default value of table from `None` to `assets` to eliminate the need for this `if` statement.